components/Inventory/InventoryMainScreen.js

```diff
@@ -1,20 +1,357 @@
+          showFastPaginationControls
+          numberOfItemsPerPageList={numberOfItemsPerPageList}
+          numberOfItemsPerPage={numberOfItemsPerPage}
+          onItemsPerPageChange={onItemsPerPageChange}
+          selectPageDropdownLabel={"Rows per page"}
+        />
+      </DataTable>
+      <Portal>
+        <Dialog
+          style={{
+            backgroundColor: "#FFFFFF",
+            width: moderateScale(300),
+            alignSelf: "center",
+          }}
+          visible={visible}
+          onDismiss={() => setVisible(false)}
+        >
+          <Dialog.Title>{title}</Dialog.Title>
+          <Dialog.Content>
+            {!isUpdate && (
+              <TextInput
+                mode="outlined"
+                label="Name"
+                style={{
+                  //width: moderateScale(50),
+                  marginVertical: 5,
+                  backgroundColor: "#FFFFFF",
+                }}
+                activeUnderlineColor={colours.ORANGE_WEB}
+                value={name}
+                onChangeText={(text) => setName(text)}
+              />
+            )}
+            <TextInput
+              mode="outlined"
+              label="stock"
+              style={{
+                //width: moderateScale(50),
+                marginVertical: 5,
+                backgroundColor: "#FFFFFF",
+              }}
+              activeUnderlineColor={colours.ORANGE_WEB}
+              keyboardType="number-pad"
+              value={stock}
+              onChangeText={(text) => setStock(text)}
+            />
+
+            <TextInput
+              mode="outlined"
+              label="MSQ"
+              style={{
+                //width: moderateScale(50),
+                marginVertical: 5,
+                backgroundColor: "#FFFFFF",
+              }}
+              activeUnderlineColor={colours.ORANGE_WEB}
+              keyboardType="number-pad"
+              value={MSQ}
+              onChangeText={(text) => setMSQ(text)}
+            />
+
+            <TextInput
+              mode="outlined"
+              label="Inventory"
+              style={{
+                //width: moderateScale(300),
+                marginVertical: 5,
+                backgroundColor: "#FFFFFF",
+              }}
+              activeUnderlineColor={colours.ORANGE_WEB}
+              keyboardType="number-pad"
+              value={inventoryStock}
+              onChangeText={(text) => setInventoryStock(text)}
+            />
+          </Dialog.Content>
+          <Dialog.Actions>
+            <Button
+              textColor={colours.OXFORD_BLUE}
+              onPress={() => setVisible(false)}
+            >
+              Cancel
+            </Button>
+            {isUpdate ? (
+              <>
+                <Button
+                  textColor={colours.ANTIQUE_RUBY}
+                  onPress={() =>
+                    setSnackbar(true, "Are you sure you want to delete", true)
```

Comment:
Cek, ovde settamo snackbar na are you sure you want to delete? Al ne bi triba bit modal? I fali upitnik